src/UI/Panels/ClipboardPanel.cs

```diff
@@ -93,21 +151,60 @@ protected override void ConstructPanelContent()
             // Clear clipboard button
             UniverseLib.UI.Models.ButtonRef clearButton = UIFactory.CreateButton(firstRow, "ClearPasteButton", "Clear Clipboard");
             UIFactory.SetLayoutElement(clearButton.Component.gameObject, minWidth: 120, minHeight: 25, flexibleWidth: 0);
-            clearButton.OnClick += () => Copy(null);
-
-            // Current Paste info row
-            GameObject currentPasteHolder = UIFactory.CreateHorizontalGroup(ContentRoot, "SecondRow", false, false, true, true, 0,
-                new(2, 2, 2, 2), childAlignment: TextAnchor.UpperCenter);
-
-            // Actual current paste info label
-            CurrentPasteLabel = UIFactory.CreateLabel(currentPasteHolder, "CurrentPasteInfo", "not set", TextAnchor.UpperLeft);
-            UIFactory.SetLayoutElement(CurrentPasteLabel.gameObject, minHeight: 25, minWidth: 100, flexibleWidth: 999, flexibleHeight: 999);
-            UpdateCurrentPasteInfo();
-
-            // Inspect button
-            UniverseLib.UI.Models.ButtonRef inspectButton = UIFactory.CreateButton(currentPasteHolder, "InspectButton", "Inspect");
+            clearButton.OnClick += ClearClipboard;
+            
+            // Inspect Selected button
+            UniverseLib.UI.Models.ButtonRef inspectButton = UIFactory.CreateButton(firstRow, "InspectButton", "Inspect");
             UIFactory.SetLayoutElement(inspectButton.Component.gameObject, minHeight: 25, flexibleHeight: 0, minWidth: 80, flexibleWidth: 0);
             inspectButton.OnClick += InspectClipboard;
+            
+            // Add New button
+            UniverseLib.UI.Models.ButtonRef addNewButton = UIFactory.CreateButton(firstRow, "AddNewButton", "Add");
```

Comment:
Is this a test code? If so, please remove it.

Reply:
This adds the button in the clipboard that actually adds a new entry. I can send a screen shot of what the clipboard looks like now.